change/@azure-msal-node-30bda6e0-6f74-44e0-981f-a7a155ef23ad.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fixed msal-node HttpClient server error bug",
```

Comment:
Can you please log a GitHub issue for this (and an equivalent issue in ADO - this is the JS team's work item pattern and you can continue to follow it - CC @peterzenz ). Having a tracking issue will ensure that the next release explicitly mentions this fix and make this searchable. 

Also please check and see if other HttpClient implementations (browser, angular etc.) suffer from this issue, so that we can discuss with JS team.

Reply:
We do not need a Github Issue for internal work or if it's tracked in ADO. The release notes will mention this change due to the changefile here and link the PR (provided you include the PR number in the comment field)

Reply:
I forgot to add the PR number to the comment field. I'll create an ADO item for the JS team as well.